src/Microsoft.ML.ImageAnalytics/MLImage.cs

```diff
@@ -0,0 +1,441 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using SkiaSharp;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.ML.Data
+{
+    /// <summary>
+    /// Provide interfaces for imaging operations.
+    /// </summary>
+    public class MLImage : IDisposable
+    {
+        private SKBitmap _image;
+
+        /// <summary>
+        /// Create a new MLImage instance from a stream.
+        /// </summary>
+        /// <param name="imageStream">The stream to create the image from.</param>
```

Comment:
Make sure to cover (and potentially test for):
1. Does the stream need to remain open after calling this API?
2. Does the stream need to be writable?  Seekable?
3. Does the stream need to contain only an image, or can it just point to the start of an image with other data before / after.

Reply:
I have added a test case for all these scenarios. 

> Does the stream need to remain open after calling this API?

No. After finishing creating the image, the stream is not used and doesn't have to remain opened. 

> Does the stream need to be writable? Seekable?

No. The stream can be non-seekable and non-writable.

> Does the stream need to contain only an image, or can it just point to the start of an image with other data before / after.

No. The steam needs to point at the beginning of the image data. The stream can have more data before and after the image data.